### DISCUSSION TITLE
❓  Help   How to classify an 20ms audio ?

### DISCUSSION THREAD

**Discussion by 649459021** [snakers4/silero-vad#92] [Q&A] [answered]
I just want to classify an 20ms audio whether there are people talking.
Could you give me some examples?
Thanks

**Comment by snakers4** [comment, marked as answer]
Hi,

20ms is a too small of an audio chunk size for our VAD - https://github.com/snakers4/silero-vad#how-vad-works
For very small chunks - please use WebRTC VAD

**Reply by 649459021** [reply]
Or maybe longer audio.
I just want to know how to classify an audio.

**Reply by snakers4** [reply]
Then just split audio into some manageable chunks (i.e. several minutes), batch them, run though the network and plot the results

You will have to fiddle a bit with thresholds to produce a simple reliable 1/0 classifier, but I guess some naive approach like a percentage of audio over certain medium threshold will work

You can push this into examples afterwards
